src/Collection/CollectionBuilder.php

```diff
@@ -71,6 +71,17 @@ public function __construct($commandFile)
     }
 
     /**
+     * @param int $verbosityLevel
+     */
+    public function setVerbosityLevel($verbosityLevel)
+    {
+        $this->verbosityLevel = $verbosityLevel;
+        $this->currentTask->setVerbosityLevel($verbosityLevel);
```

Comment:
What I really want is to set the verbosity for all tasks belonging to this collection, but I couldn't see how to access them all. $this->collection is null even when $this->currentTask is not.